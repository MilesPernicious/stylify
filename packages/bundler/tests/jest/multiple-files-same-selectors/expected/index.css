.c{
	border: 1px solid transparent
}
.d{
	margin-left: 24px
}
.e{
	font-family: arial
}
.f{
	display: block
}
.g{
	display: flex
}
.h{
	justify-content: center
}
.i{
	margin-top: 12px
}
.j{
	color: blue
}

@media (min-width: 768px) {
.a{
	color: red
}
}

@media (min-width: 1024px) {
.b{
	color: green
}
}
