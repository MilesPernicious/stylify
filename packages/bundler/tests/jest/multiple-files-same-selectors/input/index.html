<div class="border:1px_solid_transparent margin-left:24px font-family:arial display:block display:flex justify-content:center">
	<div class="margin-top:12px color:blue md:color:red lg:color:green"></div>
</div>
