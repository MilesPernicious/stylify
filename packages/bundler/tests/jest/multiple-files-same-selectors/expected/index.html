<div class="c d e f g h">
	<div class="i j a b"></div>
</div>
